Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/ImportController.java

```diff
@@ -277,10 +291,11 @@ private void uploadFileFromURL(String fileUrl) throws ExceptionImport, Exception
                     READ_TIMEOUT);
             InputStream targetStream = new FileInputStream(testData);
 
-            media = new MediaImpl(testData.getName(), targetStream, Charset.forName("UTF-8"));
+	    media = new MediaImpl(testData.getName(), targetStream, Charset.forName("UTF-8"),
```

Comment:
[Minor] Though existing code, in all files you changed in this commit, suggest utilising StandardCharsets.UTF_8 constant rather than string literal "UTF-8".

Reply:
Done